trunk/Admin/AdminQuickbarAdmin.php

```diff
@@ -230,13 +232,68 @@ public function getLoopPosts( $postType, $posts ) {
                 'permalink' => $permalink,
                 'hasSwift' => $this->hasSwift,
                 'activeClass' => $activeClass,
+                'languageFlag' => $languageFlag,
+            ] );
+            $output .= $template->getRendered();
+        }
+
+        return $output;
+    }
+
+    /**
+     * @param $post
+     * @return string
+     */
+    public function getLanguageFlag( $post ) {
+        if ( !defined( 'ICL_LANGUAGE_CODE' ) ) {
+            return '';
+        }
+
+        global $sitepress;
+        $wpmlLanguageInfo = apply_filters( 'wpml_post_language_details', null, $post->ID );
+        $languageCode = $wpmlLanguageInfo['language_code'];
+        $flagUrl = $sitepress->get_flag_url( $languageCode );
+
+        $template = new Template( self::PartialDir . '/language-flag.php', [
+            'flagUrl' => $flagUrl,
+            'alt' => $wpmlLanguageInfo['display_name'],
+            'languageCode' => $languageCode,
+        ] );
+
+        return $template->getRendered();
+    }
+
+    /**
+     * Renders all wpml language-flags
+     *
+     * @return string
+     */
+    public function renderAllLanguageFlags() {
+        $output = '';
+
+        if ( !defined( 'ICL_LANGUAGE_CODE' ) ) {
+            return '';
+        }
+
+        $wpmlLanguages = apply_filters( 'wpml_active_languages', null );
+
+        if (empty($wpmlLanguages)) {
```

Comment:
Apply spacing

Reply:
You could also combine lines 274 and 280 like this
```php
if ( !defined( 'ICL_LANGUAGE_CODE' ) || empty( $wpmlLanguages = apply_filters( 'wpml_active_languages', null ) ) ) {
```

Reply:
done